frontend/README.md

```diff
@@ -30,16 +30,40 @@ DO
 
 DON'T DO
 * Do not loose time with build process, just use create-react-app `yarn start`
-* Do not use Docker, Vagrant... we must be able to run the api only with `yarn start`
+* Do not use Docker, Vagrant... we must be able to run the app only with `yarn start`
 * Do not loose time with amazing CSS, be basic
 
-## The Test
+## Domain
 
-> For the backend data endpoints, use  `http://localhost:8080` in your frontend code
+The app is supposed to display a list of surveys and the aggregated answers to
+those. A survey is composed of a name and a code. For a given survey, aggregated
+answers are available. There are 3 types of questions:
+
+* QCM (multiple choice question)
+* Numeric
+* Date
+
+For each type of question, the aggregated answers have a different format:
+
+* QCM: the aggregated answers give the number of response for each available
+  QCM answer.
+* Numeric: the aggregated answers give the average of all answers
+* Date: the aggregated answers give the list of dates of all answers
+
+## Fake API
+
+To simulate a REST like API, copy the `api` folder into the `public/`
```

Comment:
self note: it seems create-react-app offers a way to customize init, this might be handy to replace that error prone step.